<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewFileDialogClass</class>
 <widget class="QDialog" name="CreateNewFileDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建文件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="mainLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>文件名</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="savePathLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="savePathLabel">
       <property name="text">
        <string>存储路径</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="childFileTypesComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="childFileTypeLabel">
       <property name="text">
        <string>子文件类型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fileTypeLabel">
       <property name="text">
        <string>文件类型</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="fileTypeSuffix">
       <property name="text">
        <string>suffix</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="fileTypeComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="fileNameLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="zxWidget" native="true">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="zxChaptersLabel">
          <property name="text">
           <string>关卡章节</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="zxChaptersComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="zxLevelsLabel">
          <property name="text">
           <string>关卡选择</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="zxLevelsComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="zxDiffcultiesLabel">
          <property name="text">
           <string>推荐难度</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="zxDiffcultiesComboBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomButtonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmPushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
